packages/google-cloud-discoveryengine/docs/index.rst

```diff
@@ -3,9 +3,17 @@
 .. include:: multiprocessing.rst
 
 This package includes clients for multiple versions of Discovery Engine API.
-By default, you will get version ``discoveryengine_v1``.
+By default, you will get version ``discoveryengine_v1beta``.
```

Comment:
This change is incorrect.

Reply:
The default version is listed as v1beta [here](https://github.com/googleapis/google-cloud-python/blob/81479f91ff63fdaae5142573721a1e8d433f8d46/packages/google-cloud-discoveryengine/.repo-metadata.json#L14) but it should be `v1`

Reply:
I filed https://github.com/googleapis/google-cloud-python/issues/11799. I'm going to unblock this PR as we will be tracking the improvement in https://github.com/googleapis/google-cloud-python/issues/11799